### ISSUE TITLE
[FEATURE REQ] Sound effects

### ISSUE THREAD

**Issue by greg571** [hydrafpv/roadmap#16]
Can we add a sound for when a time is read by the timer? If another pilots time is being read off in voice announcement it would be nice if a beep or other sound could play at the same time as a voice announcement, so the pilot who just made a lap knows it took their time and will read it after it's done reading the pilot before thems time. Also some kind of a buzzer if the timer reads a lap that is under the minimum lap time. This helps race directors know if they need to pay attention to a pilot issue near the start gate. Both features of other software that we are use to haveing.

**Comment by DownRightFPV** [comment]
Additionally, position announcements when passing the timer are nice to have:

"(current announcement when passing the timer) in 1st place", etc

**Comment by greg571** [comment]
Agreed, I forgot to add that we could use position announcements to go along with the lap time announcement.
